modules/sdpolicy-am-cluster-self-service.adoc

```diff
@@ -6,6 +6,6 @@
 [id="cluster-self-service_{context}"]
 = Cluster self-service
 
-Customers can create, scale, and delete their clusters from {cluster-manage-url}, provided that they have pre-purchased the necessary subscriptions.
+Customers can create, scale, and delete their clusters from {cluster-manager-url}, provided that they have pre-purchased the necessary subscriptions.
```

Comment:
In place of "pre-purchase," would "purchase" work? Unless it really needs to be purchased first, I don't think you get anything from "pre-purchase."